models/chain_info/chain_ids.sql

```diff
@@ -0,0 +1,636 @@
+{{
+  config(
+        schema = 'chain_info',
+        alias='chain_ids',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
```

Comment:
this sounds like metadata we should mostly use in spells for now (I'd like us to structure a bit better how we expose these kinds of datasets, also e.g. ABI & contracts), do we want to expose this in the UI now?

Reply:
I don't think it's a major issue if these kinds of tables stay inside e.g. a project schema, but if we create a new sector for `chain_info` I'd prefer to have a more structured approach to how we present this on the UI (without blocking this PR for it, hence I propose let's just hide it)

Reply:
Yeah, we can hide it for now. Some bridges use a different chain id structure as well, so there could be multiple chain id formats to consider later on.